app/handlers/markdown_handler.rb

```diff
@@ -0,0 +1,33 @@
+class MarkdownHandler
```

Comment:
I was totally riffing off http://www.lugolabs.com/articles/18-render-markdown-views-with-redcarpet-and-pygment-in-rails and https://www.codefellows.org/blog/how-to-create-a-markdown-friendly-blog-in-a-rails-app

I am not at all attached to the name. Is there a better convention for where to put extensions for template handlers? I suspect the name is derivative of `ActionView::Template.register_template_handler`